#id of the motor / actuator
int8    id

#setpoint of the motor / actuator system(could be RPM or position)
float32 setpoint

#desired acceleration of the motor / actuator
float32 acceleration
